doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -14,38 +14,58 @@
 
```

Comment:
We need a Synopsis which describes the "Framework" or "Path" to Certification for this RC1 Ch05 chapter, "NFVI E2E C&V Framework Requirements".

More specifically, the synopsis will mention that the document contains the Tools, Process, Steps, and Requirements for NFVI Cert Badges for NFVI Compliance, NFVI Validation, and NFVI Performance, with the pre-requisite RI delivered per RM and RA1 specifications.  

The purpose of the synopsis would be for the user to understand the Framework and Requirements, then to find more details in the document.

Sample synopsis:

_Ensure Reference Implementation (RI) of CNTT Reference Model (RM) and CNTT Reference Architecture (RA) meets industry driven quality assurance standards for compliance, verification and validation. The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.

For the purposes of this chapter, NFVI+VNF testing will be performed for Verification, Validations, ultimately leading to Certifications indicating a measured performance of the adherence to, and demonstrated proficiency with, all aspects of software delivery including but no limited to planning, logistics for communication, and testing of installed, or implemented, NFVI.  Similarily, VNFs will be checked for Compliance and Validations against the RM and RA requirements ensuring VNF instantiation, stability, and successful removal of the VNF from the implementation of the stated architecture. The additional VNF testing will verify that the developed VNF meets prescribed RM/RA infrastructure standards.

In summary, NFVI+VNF testing will be performed for Verification and Validation purposes.

- Verification will be used to indicate conformance to design requirement specifications. Activities involved Reviews and Walk-Throughs to ensure the NFVI is delivered per implementation specifications.

- Validations is used to indicate testing performed to confirm the actual output of a product meets the expected, or desired outcome, or behavior.  

**All Terms utilized throughout this chapter are intended to align with CVC definitions, and their use through CVC documentation, guidelines, and standards.**

...This is the same content I wrote for Ch 1 of RC-1.  Applicable here, so feel free to use all (or some), and also consider adding the following:  "This chapter will outline the Requirements, Process, and Automation, needed to deliver NFVI Certification."  --> This additional statement will setup the reader for the text and Into paragraph next.

Reply:
Added

Reply:
Let me know if we can resolve this.

Reply:
Looks good.  Closing/Resolving now.